frappe/integrations/doctype/dropbox_settings/dropbox_settings.py

```diff
@@ -86,7 +86,7 @@ def backup_to_dropbox():
 		access_token = generate_oauth2_access_token_from_oauth1_token(dropbox_settings)
 
 		if not access_token.get('oauth2_token'):
-			return
+			return None, None
```

Comment:
Instead of None, pass 'Failed backup upload', 'Access Token No exists! Please generate the access token for Dropbox'.

If we pass None, then it will send email as backup loaded successfully!
https://github.com/manassolanki/frappe/blob/fb47a6f4a745813032bca14208b03efb8792cbe0/frappe/integrations/doctype/dropbox_settings/dropbox_settings.py#L46